packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -534,61 +573,28 @@ export default class Meetings extends WebexPlugin {
      * @memberof Meetings
      */
     destroy(meeting, reason) {
-      if (this.config.autoUploadLogs) {
-        LoggerProxy.logger.info('Meetings:index#destroy --> Meeting destroyed, uploading logs');
-        const callStart = meeting.locusInfo && meeting.locusInfo.fullState ? meeting.locusInfo.fullState.lastActive : undefined;
-
-        this.uploadLogs({
-          callStart,
-          correlationId: meeting.correlationId,
-          feedbackId: meeting.correlationId,
-          locusId: meeting.locusId
-        })
-          .then((uploadResult) => {
-            LoggerProxy.logger.info('Meetings:index#destroy --> Upload logs for meeting completed.', uploadResult);
-            Trigger.trigger(
-              this,
-              {
-                file: 'meetings',
-                function: 'destroy'
-              },
-              EVENT_TRIGGERS.MEETING_LOG_UPLOAD_SUCCESS,
-              {
-                meetingId: meeting.id,
-                details: uploadResult
-              }
-            );
-          })
-          .catch((uploadError) => {
-            LoggerProxy.logger.error('Meetings:index#destroy --> Unable to upload logs for meeting', uploadError);
-            Trigger.trigger(
-              this,
-              {
-                file: 'meetings',
-                function: 'destroy'
-              },
-              EVENT_TRIGGERS.MEETING_LOG_UPLOAD_FAILURE,
-              {
-                meetingId: meeting.id,
-                reason: uploadError
-              }
-            );
-          });
-      }
-
-      this.meetingCollection.delete(meeting.id);
-      Trigger.trigger(
-        this,
-        {
-          file: 'meetings',
-          function: 'destroy'
-        },
-        EVENT_TRIGGERS.MEETING_REMOVED,
-        {
-          meetingId: meeting.id,
-          reason
-        }
-      );
+      return this.uploadLogs({
```

Comment:
Why are we still uploading logs on destroy? I thought this PR was to change to upload logs on leave.

Reply:
actually we want to upload logs on both leave and destroy as user on leave will refresh the browser or close rather then wait for destroy so we are uploading it on both instance but there could be 2 instance on upload some time which is fine for now

Reply:
Ok, I've updated the PR title to reflect this.